setup.py

```diff
@@ -6,21 +6,44 @@
 
 setup(
     name="Open edX",
-    version="0.2",
-    install_requires=['distribute'],
+    version="0.3",
```

Comment:
Should we bump the minor version every time we add entry points? I think so, since this is more than just a patch.

Also, when will this become version 1.0? It seems like it might be out of beta already. :-)